amethyst_test/Cargo.toml

```diff
@@ -17,7 +17,9 @@ amethyst = { path = "..", version = "0.13.0" }
 derivative = "1.0"
 derive-new = "0.5"
 derive_deref = "1.1.0"
+humantime = "1.3.0"
```

Comment:
Is this additional dependency necessary? I don't see it being used.

Reply:
yeaps, it's [here](https://github.com/amethyst/amethyst/pull/1984/files#diff-4aaf838744cff2933e207ed6e7978c36R75),

That said, knowing it's 10 seconds, we could hard code the message. Would you prefer that?

Reply:
I would prefer that, it doesn't make sense to bring in the dep for that one call imo

Reply:
done done :upside_down_face: